accounts/admin.py

```diff
@@ -0,0 +1,3 @@
+from django.contrib import admin
```

Comment:
Remove unused files from the app - if you don't need to register anything in the django admin, you can safely delete this.